.gitignore

```diff
@@ -5,22 +5,20 @@
 
 # cargo
 target
-*/target
-*.rs.bk
 Cargo.lock
 
 # fuzz testing
-exonum/fuzz/target
 exonum/fuzz/corpus
 exonum/fuzz/artifacts
 
+# node
+node_modules
+
 # node (for cspell)
-/node_modules
 /npm-debug.log
```

Comment:
It could make sense to make `npm-debug.log` directory-independent, same as `node_modules`.